plugins/azure/frontdoor/frontDoorManagedIdentity.js

```diff
@@ -0,0 +1,60 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Front Door Managed Identity Enabled',
+    category: 'Front Door',
+    domain: 'Content Delivery',
+    severity: 'Medium',
+    description: 'Ensures that Azure Front Door have managed identity enabled.',
+    more_info: 'Enabling managed identities eliminate the need for developers having to manage credentials by providing an identity for the Azure resource in Azure AD and using it to obtain Azure Active Directory (Azure AD) tokens.',
+    recommended_action: 'Modify the Front Door classic profile and add managed identity.',
+    link: 'https://learn.microsoft.com/en-us/azure/frontdoor/managed-identity',
+    apis: ['profiles:list'],
+    realtime_triggers: ['microsoftcdn:profiles:write', 'microsoftcdn:profiles:delete'],
+
+    run: function(cache, settings, callback) {
+        const results = [];
+        const source = {};
+        const locations = helpers.locations(settings.govcloud);
+        async.each(locations.profiles, (location, rcb) => {
+            const profiles = helpers.addSource(cache, source,
+                ['profiles', 'list', location]);
+
+            if (!profiles) return rcb();
+
+            if (profiles.err || !profiles.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query Front Door profiles: ' + helpers.addError(profiles), location);
+                return rcb();
+            }
+
+            if (!profiles.data.length) {
+                helpers.addResult(results, 0, 'No existing Azure Front Door profiles found', location);
+                return rcb();
+            }
+
+            var frontDoorProfile = false;
+            profiles.data.forEach(function(profile) {
+                if (!profile.id || profile.kind!='frontdoor') return;
+                
```

Comment:
According to the https://learn.microsoft.com/en-us/azure/frontdoor/managed-identity#prerequisites 
Before you can set up managed identity for Azure Front Door, you must have an Azure Front Door Standard or Premium profile created

can we also add check for premium and standard?

Reply:
in the list call we only get standard and premium forntdoor